spec/filepreviews/utils_spec.rb

```diff
@@ -24,4 +24,26 @@ class Kawaii
       expect(Kawaii.new.extract_size(size)).to eq('320x240')
     end
   end
+
+  describe '.validate_pages' do
+    context 'when called with range parameters (1-3)' do
+      it 'validates page parameters' do
+        expect(Kawaii.new.validate_pages('1-3'))
+          .to eq('1-3')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.